client/public/theme.css

```diff
@@ -25,15 +20,32 @@ h1, h2, h3, h4, h5, h6 {
 
 .p-input-icon-left > i,
 .p-input-icon-right > i {
-  margin-top: -12px!important;
+  margin-top: -12px !important;
 }
 
 .p-input-icon-left > .p-inputtext {
-    padding-left: calc(1.25rem + 24px);
+  padding-left: calc(1.25rem + 24px);
 }
 
 .p-input-icon-right > .p-inputtext {
-    padding-right: calc(1.25rem + 24px);
+  padding-right: calc(1.25rem + 24px);
+}
+
+.p-toast .p-toast-message .p-toast-message-content {
+  border-width: 0!important;
+}
+
+.p-toast .p-toast-message,
+.p-message {
+  border: 0 none!important;
+}
+
+.p-toast-message-icon {
+  height: 1em!important;
+}
```

Comment:
space before !important, as you did above

Reply:
I can't remember putting this space, but found this telling I shouldn't:

https://stackoverflow.com/a/31143078

Reply:
"is not required", but should be added, as per conventions

https://developer.mozilla.org/en-US/docs/Web/CSS/important

Reply:
Alright, you got this point